content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -16,16 +16,29 @@ This page describes the CoreDNS upgrade process and how to install kube-dns inst
 
 {{% capture steps %}}
 
-## Installing kube-dns with kubeadm
+## About CoreDNS
 
-In Kubernetes 1.11, [CoreDNS](https://coredns.io) has graduated to General Availability (GA)
+[CoreDNS](https://coredns.io) is a flexible, extensible authoritative DNS server that can run on a Kubernetes cluster.
+The CoreDNS project is hosted, like Kubernetes, by the [CNCF.](http://www.cncf.io)
+
+You can use CoreDNS instead of kube-dns in your cluster by directly replacing kube-dns in an existing
+deployment, or using tools like kubeadm that will deploy and upgrade the cluster for you.
+
+## Installing CoreDNS
+
+For manual deployment or replacement of kube-dns, refer to the documentation at the
+[CoreDNS GitHub project.](https://github.com/coredns/deployment/tree/master/kubernetes)
```

Comment:
I'm guessing we don't have time to fix up for 1.11 -- but linking out to docs in a GH repo is sub-optimal UX. Or is this a deliberate ploy to drive users to kubeadm? Maybe create an issue to discuss?

Reply:
@Bradamant3 : Would it be better to link to that page : https://coredns.io/plugins/kubernetes ?
It is a documentation page dedicated to Kubernetes on CoreDNS.io website.

Reply:
leave it as is, then.